pytorch_lightning/utilities/argparse.py

```diff
@@ -312,3 +313,22 @@ def _precision_allowed_type(x: Union[int, str]) -> Union[int, str]:
         return int(x)
     except ValueError:
         return x
+
+
+def _defaults_from_env_vars(fn: Callable) -> Callable:
+    @wraps(fn)
```

Comment:
Should we keep the docstring?

```suggestion
def _defaults_from_env_vars(fn: Callable) -> Callable:
    """Decorator for :class:`~pytorch_lightning.trainer.trainer.Trainer` methods for which input arguments should
    be moved automatically to the correct device."""

    @wraps(fn)
```

Reply:
The docstring is wrong.